src/axolotl/utils/schemas/config.py

```diff
@@ -664,6 +664,44 @@ class AxolotlInputConfig(
 
     llama4_linearized_experts: bool | None = None
 
+    # MoE aux-loss-free (AFB) toggles
+    moe_balance_type: Literal["gshard", "noaux_tc"] | None = Field(
```

Comment:
Since this is integration, we can add these as integration config instead of setting in the global config.

Here is an example https://github.com/axolotl-ai-cloud/axolotl/blob/main/src/axolotl/integrations/kd/args.py

Then, these args will be loaded if a user specifies the plugin.